core/server/proxy/src/main/java/alluxio/proxy/s3/CompleteMultipartUploadHandler.java

```diff
@@ -200,6 +208,7 @@ public class CompleteMultipartUploadTask implements
     private final String mBody;
     private final boolean mMultipartCleanerEnabled = Configuration.getBoolean(
         PropertyKey.PROXY_S3_MULTIPART_UPLOAD_CLEANER_ENABLED);
+    private AlluxioURI mMultipartTemporaryDir;
```

Comment:
Sry, I don't find anywhere uses this variable

Reply:
removed, thanks for catching